README.md

```diff
@@ -102,23 +95,34 @@ Other make targets include (more info below):
+* `make clean`: removes the binary build output and `splunk-operator` container image example `make docker-push IMG=docker.io/splunk/splunk-operator:<tag name>`
 * `make run`: runs the splunk operator locally, monitoring the Kubernetes cluster configured in your current `kubectl` context
 * `make fmt`: runs `go fmt` on all `*.go` source files in this project
 * `make lint`: runs the `golint` utility on all `*.go` source files in this project
-* `make bundle-build`: generates `splunk-operator-bundle` bundle container image for OLM example `make bundle-build IMAGE_TAG_BASE=docker.io/splunk/splunk-operator VERSION=1.1.0 IMG=docker.io/splunk/splunk-operator:1.1.0`
-* `make bundle-push`: push OLM bundle docker image to given repository example `make bundle-push IMAGE_TAG_BASE=docker.io/splunk/splunk-operator VERSION=1.1.0 IMG=docker.io/splunk/splunk-operator:1.1.0`
-* `make catalog-build`: generates `splunk-operator-catalog` catalog container image example `make catalog-build IMAGE_TAG_BASE=docker.io/splunk/splunk-operator VERSION=1.1.0 IMG=docker.io/splunk/splunk-operator:1.1.0`
-* `make catalog-push`: push catalog docker image to given repository example`make catalog-push IMAGE_TAG_BASE=docker.io/splunk/splunk-operator VERSION=1.1.0 IMG=docker.io/splunk/splunk-operator:1.1.0`
+* `make bundle-build`: generates `splunk-operator-bundle` bundle container image for OLM example `make bundle-build IMAGE_TAG_BASE=docker.io/splunk/splunk-operator VERSION=<tag name> IMG=docker.io/splunk/splunk-operator:<tag name>`
+* `make bundle-push`: push OLM bundle docker image to given repository example `make bundle-push IMAGE_TAG_BASE=docker.io/splunk/splunk-operator VERSION=<tag name>IMG=docker.io/splunk/splunk-operator:<tag name>`
+* `make catalog-build`: generates `splunk-operator-catalog` catalog container image example `make catalog-build IMAGE_TAG_BASE=docker.io/splunk/splunk-operator VERSION=<tag name> IMG=docker.io/splunk/splunk-operator:<tag name>`
+* `make catalog-push`: push catalog docker image to given repository example`make catalog-push IMAGE_TAG_BASE=docker.io/splunk/splunk-operator VERSION=<tag name>IMG=docker.io/splunk/splunk-operator:<tag name>`
 
-## Running the Splunk Operator
-
-Ensure that you have the Custom Resource Definitions and RBAC policies installed in your cluster:
+## Deploying the Splunk Operator
+`make deploy` command will deploy all the necessery RBAC policies, services ,config maps, deployment necessary to run splunk operator. opeartor will be installed in `splunk-operator` namespace. if `splunk-operator` namespace do not exist , it will create the namespace.  By default `make deploy` will install operator clusterwide. operator will watch all the namespaces for any splunk entrprise custom resources.
 
 ```shell
-make install
+make deploy IMG=docker.io/splunk/splunk-operator:<tag name>
+```
+
+If user wants to deploy operator for specific namespace then they has to pass `WATCH_NAMESPACE` parameter to `make deploy` command 
```

Comment:
change "they has to pass" to "they must pass"

Reply:
Fixed